android/src/main/java/com/gettipsi/stripe/GoogleApiPayFlowImpl.java

```diff
@@ -98,17 +99,26 @@ private PaymentMethodTokenizationParameters createPaymentMethodTokenizationParam
   private PaymentDataRequest createPaymentDataRequest(ReadableMap payParams) {
     final String estimatedTotalPrice = payParams.getString(TOTAL_PRICE);
     final String currencyCode = payParams.getString(CURRENCY_CODE);
-    final boolean billingAddressRequired = Converters.getValue(payParams, BILLING_ADDRESS_REQUIRED, false);
+    final Boolean billingAddressRequired = Converters.getValue(payParams, BILLING_ADDRESS_REQUIRED, false);
     final Boolean shippingAddressRequired = Converters.getValue(payParams, SHIPPING_ADDRESS_REQUIRED, false);
+    final Boolean phoneNumberRequired = Converters.getValue(payParams, PHONE_NUMBER_REQUIRED, false);
```

Comment:
Minor stuff: It's better for these 3 `Boolean` to be `boolean` to avoid potential null unboxing in future.